gridded_etl_tools/utils/attributes.py

```diff
@@ -182,6 +187,13 @@ def hindcast_steps(self) -> list[int]:
         """To be overwritten by actual hindcast datasets"""
         return list(None)
 
+    @property
```

Comment:
I think every dataset class will want to always have this same property, so we would want this to be a class property. Related to this, I posted a comment that maybe all of the attributes besides store should be class properties https://github.com/Arbol-Project/gridded-etl-tools/pull/91#issuecomment-1808791840

Reply:
By a class property you mean use the system Chris has set up in #91 ? Makes sense to me but let’s get this PR merged and then I’ll work on integrating that PR into our managers and attributes.